profile.go

```diff
@@ -235,6 +235,8 @@ fetching may be degraded.
 		Transform: func(c *Config) error {
 			c.Experimental.Libp2pStreamMounting = true
 			c.Experimental.StorageClientEnabled = true
+			c.Experimental.HostsSyncEnabled = true
+			c.Experimental.HostsSyncMode = DefaultHostsSyncMode
```

Comment:
would this original setting do the work?
https://github.com/TRON-US/go-btfs-config/commit/dfd27a896bbda6d80851f2c0496365364c59ef23#diff-33a47ea8c5a16ace99278226b31e7076R88-R90

Reply:
No because it's only on init. If someone switches it off but wants to become a renter, it should be turned on again.

Reply:
I see, make sense.